core/src/test/scala/integration/kafka/api/PlaintextConsumerTest.scala

```diff
@@ -794,9 +797,12 @@ class PlaintextConsumerTest extends BaseConsumerTest {
     awaitAssignment(consumer, partitions.toSet)
 
     val producer = createProducer()
-    val producerRecords = partitions.flatMap(sendRecords(producer, numRecords = 15, _))
+    // it'll send 30 records to each topic partition (3 topics and 30 partitions each topic)
+    val producerRecords = partitions.flatMap(sendRecords(producer, numRecords = partitionCount, _, needCurrentTime = true))
+
     val consumerRecords = consumeRecords(consumer, producerRecords.size)
 
+    // the expected records should be 30 records * 3 topics * 30 partitions = 2700
```

Comment:
What is the purpose of this and the other comment?

Reply:
Let me remove it.